portal/static/js/profile.js

```diff
@@ -2471,32 +2471,55 @@
                 });
                 return content;
             },
+            getConsentEditDisplayIconHTML: function(item, targetElementId) {
+                if (!item) {
+                    return "";
+                }
+                return "&nbsp;&nbsp;<a data-toggle='modal' data-target='#{targetElementId}' data-orgId='{orgId}' data-agreementUrl='{agreementUrl}' data-userId='{userId}' data-status='{consentStatus}' data-signed-date='{signedDate}'><span class='glyphicon glyphicon-pencil edit-icon' aria-hidden='true'></span></a>"
+                    .replace("{targetElementId}", targetElementId||"")
+                    .replace("{orgId}", item.organization_id)
+                    .replace("{agreementUrl}", item.agreement_url)
+                    .replace("{userId}", this.subjectId)
+                    .replace("{consentStatus}", this.getConsentStatusHTMLObj(item).statusText)
+                    .replace("{signedDate}", this.modules.tnthDates.formatDateString(item.acceptance_date, "d M y hh:mm:ss"));
```

Comment:
Is this in GMT? Can you include the timezone in the datetime?

Reply:
@ivan-c yeah, ivan, this is GMT.  is displayed and instructed to user as such in the consent date modal:  https://github.com/uwcirg/true_nth_usa_portal/blob/7f2a1acd692aaff131a16d62315f8e381a3cc70f/portal/templates/profile/profile_macros.html#L748
so no date/time conversion based on timezone will take place here.

Reply:
Awesome! Would you be able to submit the datetime in ISO8601 format (eg `2018-09-26T15:10:19:23Z`) to make it obvious it's GMT, since the time is now being included (and the is now significant)?